project_admin/views.py

```diff
@@ -49,5 +50,9 @@ class LoginView(FormView):
 
     def form_valid(self, form):
         token = form.cleaned_data['token']
+        req_url = ("https://www.openhumans.org/api/direct-sharing/project/?access_token={}".format(token))
+        params = {'token': token}
+        r = requests.get(req_url, params=params).json()
+        Project.objects.update_or_create(**r)
```

Comment:
This actually results in an error when a Project already exists but has updated information (e.g. a new contact email). I believe parameters should be used to find a matched object (so `id` should be sufficient), and the update is then passed as a dict to the `default` parameter, see: https://docs.djangoproject.com/en/2.0/ref/models/querysets/#update-or-create

Otherwise everything looks great! I'm going to submit a commit that fixes this issue. :)